src/components/Register/Register.js

```diff
@@ -116,7 +127,11 @@ const Register = () => {
                                 </i>
                             </label>
                         </Box>
-                        <Input id="file-upload" type="file"></Input>{' '}
+                        <Input
+                            id="file-upload"
+                            type="file"
+                            ref={chooseFile}
+                        ></Input>{' '}
```

Comment:
Components that don't have children should be self closing

```suggestion
                        <Input
                            id="file-upload"
                            type="file"
                            ref={chooseFile}
                        />
```